lib/docker.ts

```diff
@@ -103,7 +105,14 @@ class Docker {
     if (recursive) {
       params += "R";
     }
-    return this.runSafe("ls", [params, path]);
+
+    const ignoreErrors = [
```

Comment:
```suggestion
    const ignoreErrors: string[] = [
```

Reply:
Here I actually prefer the former, because the syntax is less noisy, and hence the code more readable. Other than that they're identical compile and run time.